### PR TITLE
Did some changes in the pages according to the requirement

### PR DESCRIPTION
**Pull request by anshikaa12**
>adjusted logo
>added more input fields in login page
>added rating filter in sidebar
>made featured category link to products page

### CODE REVIEW

**Review by sharathnair9999 — APPROVED** [review]
Clean code. You can merge it.

**Review by rohitrout — APPROVED** [review]
Looks good to merge

File: css/pages/login/login.html [hunk 1]
```diff
@@ -57,19 +57,31 @@
...
-            <input type="text"  placeholder="Enter Password" class="input-md all-border-input">
+            <input type="password" placeholder="Enter Password" class="input-md all-border-input"placeholder="Enter Password" class="input-md all-border-input">
+        </div>
            <div id="register-input" class="hide">
+            <h4 class="main-h4">First Name</h4>
+            <input type="text"  placeholder="Enter First name" class="input-md all-border-input">
+            <h4 class="main-h4">Last Name</h4>
+            <input type="text"  placeholder="Enter Last name" class="input-md all-border-input">
+            <h4 class="main-h4">Email</h4>
+            <input type="text"  placeholder="Enter email" class="input-md all-border-input">
```
Comment (MaruthiPothuganti) [inline css/pages/login/login.html:72]: input type of email can be used here

File: css/layout/product-sidebar.css [hunk 2]
```diff
@@ -139,29 +139,27 @@
...
-/* When the radio button is checked, add a blue background */
+
 .radio-list input:checked~.radio {
     background-color: var(--primary-color);
 }
 
-/* Create the indicator (the dot/circle - hidden when not checked) */
+
 .radio:after {
     content: "";
```
Comment (Avin008) [inline css/layout/product-sidebar.css:153]: remove empty properties

File: css/pages/login/login.html [hunk 3]
```diff
@@ -57,19 +57,31 @@
...
             <h4  class="main-h4">Re-Enter Password</h4>
-            <input type="text"  placeholder="Re-Enter Password" class="input-md all-border-input">
+            <input type="password"  placeholder="Re-Enter Password" class="input-md all-border-input">
         </div>
         </div>
 <div class="login-btn flex-col-center">
     <button class="btn-primary btn">LOGIN</button>
-    <a href="" class="prime-color-text wt-md h5-text">Forgot Password?</a>
+    <a href="" id="login-link" class="prime-color-text wt-md h5-text">Forgot Password?</a>
+    <a href="" id="register-link" class="hide prime-color-text wt-md h5-text">Already have an account? Login here</a>
```
Comment (MaruthiPothuganti) [inline css/pages/login/login.html:82]: No hyperlink to create account page

**Review by Avin008 — APPROVED** [review]
looks great good to merge

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a couple of additions. Everyhing else looks great. Good to merge

**Pull request merged**